app/_kong_plugins/header-cert-auth/index.md

```diff
@@ -36,6 +36,144 @@ search_aliases:
   - header cert auth
   - header-cert-auth
   - authentication
+
+faqs: 
+  - q: Will the client need to encrypt the message with a private key and certificate when passing the certificate in the header?
+    a: |
+      No, the client only needs to send the target's certificate encoded in a header. {{site.base_gateway}} will validate the certificate, but it requires a high level of trust that the WAF/LB is the only entrypoint to the {{site.base_gateway}} proxy. The Header Cert Auth plugin will provide an option to secure the source, but additional layers of security are always preferable. Network level security (so that {{site.base_gateway}} only accepts requests from WAF - IP allow/deny mechanisms) and application-level security (Basic Auth or Key Auth plugins to authenticate the source first) are examples of multiple layers of security that can be applied.
+  - q: How should the certificate be passed in a client request?
+    a: |
+      This depends on the format specified in the `certificate_header_format` parameter. When using `base64_encoded`, only the base64-encoded body of the certificate should be sent (excluding the `BEGIN CERTIFICATE` and `END CERTIFICATE` delimiters).  When using `url_encoded`, the entire certificate, including the `BEGIN CERTIFICATE` and `END CERTIFICATE` delimiters, should be provided.
+
+      For example, given the `certificate_header_name` of x-client-cert:
+
+      `base64_encoded`
+
+      ```bash
+      x-client-cert: MIIDbDCCAdSgAwIBAgIUa...
+      ```
+
+      `url_encoded`
+
+      ```bash
+      x-client-cert: -----BEGIN%20CERTIFICATE-----%0AMIIDbDCCAdSgAwIBAgIUa...-----END%20CERTIFICATE-----
+      ```
 ---
 
-## Overview
+The Header Cert Authentication plugin authenticates API calls by using client certificates provided in HTTP headers,
+instead of relying on traditional TLS termination.
+This approach is particularly useful in scenarios where TLS traffic is terminated outside of {{site.base_gateway}} such as at an external CDN or load balancer and the client certificate is passed along in an HTTP header for subsequent validation.
+
+{:.warning}
+> **Important:** To use this plugin, you must add [certificate authority (CA) certificates](/gateway/entities/ca-certificate/). Set them up before configuring the plugin.
+
+## How it works
+
+This plugin addresses the inability of the {{site.base_gateway}} to authenticate API calls using client certificates received in HTTP headers, rather than through traditional TLS termination. 
```

Comment:
```suggestion
This plugin lets {{site.base_gateway}} authenticate API calls using client certificates received in HTTP headers, rather than through traditional TLS termination. 
```
Rephrasing to make this a positive thing.